tests/test_beta_distributions.py

```diff
@@ -122,12 +122,3 @@ def test_christoffels_vectorization(self):
         expected = gs.array(
             [self.n_samples, self.dim, self.dim, self.dim])
         self.assertAllClose(result, expected)
-
-    def test_inner_product_matrix(self):
-        point = gs.array([1., 1.])
-        result = self.beta.metric.inner_product_matrix(point)
-        expected = gs.array([[1., -0.644934066], [-0.644934066, 1.]])
-        self.assertAllClose(result, expected)
-
-        self.assertRaises(
-            ValueError, self.metric.inner_product_matrix)
```

Comment:
Why did you remove this test? Maybe a rebase error

Reply:
No, this function does not raise an error for this case anymore, as it was not consistent with the behavior of other methods in the code.

Reply:
Ok, that’s better, but why not remove just the last assertRaises and keep the rest of the test?

Reply:
Ok!